src/wavesurfer.js

```diff
@@ -342,6 +342,17 @@ var WaveSurfer = {
     },
 
     /**
+     * Cancels the audio loading and destorys all necessary handlers
+     * and connections.
+    **/
+    cancelLoad: function (xhrObject) {
+        xhrObject.xhr.abort();
+        this.Observer.unAll();
```

Comment:
This will kill all external subscribers. I think it's not what most users would want.